server/service/src/invoice_line/stock_out_line/insert/generate.rs

```diff
@@ -105,6 +109,8 @@ fn generate_line(
         total_before_tax,
         note,
         campaign_id,
+        item_variant_id: input_item_variant_id,
+        donor_id: input_donor_id,
```

Comment:
Tangentially I think I prefer keeping the parameters of functions terse by just defining the structs and then destructuring in the function body

Reply:
In fact when there is a number of fields coming from structs with some overlap maybe I don't like destructuring at all haha